### DISCUSSION TITLE
SO and existing Elastic instance

### DISCUSSION THREAD

**Discussion by rosswakelin** [Security-Onion-Solutions/securityonion#3370] [Unsupported Versions] [answered]
We already have an instance of Elastic where we save all our syslogs, firewall logs etc., making use of the ECS parsing built into the "Basic" (i.e. free) licence level.  This is a cluster of three ingest nodes and two data nodes.  We have crafted our own NIDS sensors using Suricata and Zeek, using filebeat and logstash to send things back to the cluster.
We like the integrations in the SO, and we are trying to work out if we can we somehow install the SO environment but use our existing Elastic storage cluster instead of the SO distributed SO "search node" concept.  Does anyone have any suggestions?

Thanks

Ross

**Comment by weslambert** [comment, marked as answer]
You could try pointing the Kibana/Elastic from the manager at the remote cluster, and have the remote cluster's Logstash instance pull from Redis, or have Logstash on Security Onion send to your remote instance, but we can't really provide community support for an integration of this type at this time.
